test/functionalTest/cases/3190_notify_only_attributes_that_change/notify_only_attributes_that_change.test

```diff
@@ -0,0 +1,497 @@
+# Copyright 2019 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Notify only attributes that change for NGSIv2 POST /v2/entities, POST /v2/subscriptions, Dump and reset accumulator, Sleep 5 secs, PATCH /v2/entities/Room1/attrs, Dump and reset accumulator, Sleep 5 secs, PATCH /v2/entities/Room1/attrs, Dump and reset accumulator, Sleep 5 secs, PATCH /v2/entities/Room1/attrs, Dump and reset accumulator, Sleep 5 secs, PATCH /v2/entities/Room1/attrs, Dump and reset accumulator.
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+accumulatorStart --pretty-print localhost $LISTENER_PORT
+
+--SHELL--
+
+#
```

Comment:
It will be better to include more detail in the steps (and associated echo messages). In particular:

In step 2, please describe the detail of the subscription. For instance:

```
02. POST /v2/subscriptions, to create subscription for Room1 with cond attrs: [A, B, C] notif attrs: [B, C, D], onlyChangedAttrs: true
```



In the "Dump and reset accumulator, see notification" it would be great to show the expected notification. For instance, instead of

> 03. Dump and reset accumulator, see notification.
> ...
> 06. Dump and reset accumulator, see notification.

Use

> 03. Dump and reset accumulator, see B, C and D (as this is initial notification)
> ...
> 06. Dump and reset accumulator, see B

Reply:
fixed in f9887f1